src/content/advanced/gitops/apps/automatic_updates_appcr.md

```diff
@@ -0,0 +1,252 @@
+---
+linkTitle: Enable automatic updates in Apps
+title: Enable automatic updates in Apps
+description: Learn how to enable and configure automatic updates in Apps deployed using GitOps.
+weight: 50
+menu:
+  main:
+    identifier: advanced-gitops-apps-automatic-update
+    parent: advanced-gitops-apps
+last_review_date: 2023-02-10
+user_questions:
+  - How can I enable and configure auto update for apps deployment with GitOps?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+---
+
+This document is part of the documentation to use GitOps with Giant Swarm App Platform. You can find more information about the [App Platform in our docs](/platform-overview/app-platform/).
+
+# Enable automatic updates of an existing App
+
+Follow the below instructions to instruct Flux on how to automatically update an existing App. Automatic update means Flux will scan a remote repository and automatically update your deployed application version while also creating commits in your repository to reflect these changes.
+
+Configuring automated version updates requires a few additional Flux API objects to be defined, namely:
+
+- `ImageUpdateAutomation`, which defines the core update properties, like the git repo where the commits are to be made, commit messages structure and how to update files in the repository,
+- `ImageRepository` to configure where images are stored and new versions can be detected,
+- `ImagePolicy` to configure which of the detected new tags will be considered as possible to update to.
+- Additionally, you have to edit your existing App manifest to include a marker showing the manifest's property you want to have auto updated by Flux (usually App version property).
+
+Flux will watch for new Docker image tags for your App and use them to update the `.spec.version` field in the App CR. It will do it by pushing commits to this repository.
+
+**__Note__**, in order to use this mechanism you have to make sure that image tags of your App correspond to its version, otherwise this process will result in setting a meaningless version in the `.spec.version` field.
+
+## Example
+
+An example of an App automated updates is available in the [gitops-template repository in "WC_NAME/apps/hello-world-automatic-updates"](https://github.com/giantswarm/gitops-template/tree/main/management-clusters/MC_NAME/organizations/ORG_NAME/workload-clusters/WC_NAME/mapi/apps/hello-world-automatic-updates/).
+
+## Export environment variables
+
+**__Note__**, Management Cluster codename, Organization name, Workload Cluster name and some App-related values are needed in multiple places across this instruction, the least error prone way of providing them is by exporting them as environment variables:
```

Comment:
```suggestion
__Note__: in order to use this mechanism you have to make sure that the image tags of your App correspond to its version, otherwise this process will result in setting a meaningless version in the `.spec.version` field.

## Example

An example of an App automated update is available in the [gitops-template repository in "WC_NAME/apps/hello-world-automatic-updates"](https://github.com/giantswarm/gitops-template/tree/main/management-clusters/MC_NAME/organizations/ORG_NAME/workload-clusters/WC_NAME/mapi/apps/hello-world-automatic-updates/).

## Export environment variables

__Note__: Management Cluster codename, Organization name, Workload Cluster name and some App-related values are needed in multiple places across this instruction, the least error prone way of providing them is by exporting them as environment variables:
```